sys/linux/io_uring.txt

```diff
@@ -3,56 +3,340 @@
 
 # See http://kernel.dk/io_uring.pdf
 
-# TODO: we don't write anything to the actual uring.
 # NEED: we somehow need to write the mmap-ed range after the mmap. vring needs something similar.
```

Comment:
This is resolved as well.

Reply:
[removed](https://github.com/google/syzkaller/pull/1926/commits/9640a741e26eec6a780e5936f0407986c96a0cea#diff-2e824a90609ef73cc6319dbc406b6b5eL6)